src/Sylius/Bundle/ContentBundle/Form/Type/ActionBlockType.php

```diff
@@ -33,7 +33,20 @@ public function buildForm(FormBuilderInterface $builder, array $options = array(
             ->add('actionName', 'text', array(
                 'label' => 'sylius.form.action_block.action_name',
             ))
-        ;
+            ->add('publishable', null, array(
+                    'label' => 'sylius.form.action_block.publishable'
```

Comment:
Please fix indent to match already used one, in all files you have changed, thanks!